packages/aws-cdk/test/integ/cli/test.sh

```diff
@@ -5,6 +5,8 @@ source ${scriptdir}/common.bash
 
 header CLI Integration Tests
 
+npm run integ
```

Comment:
I'd rather the execution of the js goes into its own shell script which runs just like the other shell scripts that represent tests, rather than being a different-class citizen like this.

Reply:
So, just to be 100% clear what needs to happen here:

1. You want to remove the above change.
2. You want to add a file in the `test/integ/cli` directory called something like `test-npm-integ.sh`, with the contents:

```bash
#!/bin/bash

set -euo pipefail
scriptdir=$(cd $(dirname $0) && pwd)
source ${scriptdir}/common.bash

npm run integ
```

?

Reply:
Indeed

Reply:
Done.